pom.xml

```diff
@@ -176,7 +176,6 @@ THE SOFTWARE.
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-support</artifactId>
             <version>2.13</version>
-            <scope>test</scope>
```

Comment:
`workflow-support` was already a dependency, via `workflow-durable-task-step`. The POM formerly overrode the scope to be `test` (it would otherwise have defaulted to `compile`). Since I needed a facility introduced in 2.13, this sufficed.